src/platform/forms-system/src/js/helpers.js

```diff
@@ -82,13 +82,19 @@ export function getInactivePages(pages, data) {
 }
 
 export function createFormPageList(formConfig) {
-  return Object.keys(formConfig.chapters).reduce((pageList, chapter) => {
-    const chapterTitle = formConfig.chapters[chapter].title;
-    const pages = Object.keys(formConfig.chapters[chapter].pages).map(page => ({
-      ...formConfig.chapters[chapter].pages[page],
+  if (!formConfig?.chapters) return [];
+
+  return Object.keys(formConfig.chapters).reduce((pageList, chapterKey) => {
+    const chapter = formConfig.chapters[chapterKey];
+
+    if (!chapter?.pages) return pageList;
+
+    const chapterTitle = chapter?.title ?? formConfig.title;
```

Comment:
Neither here nor there but why use `??` instead of `||`?

Reply:
The idea I had with this logic was that if a user wanted to pass an empty string in as the title in order to have a blank space for that section, they could. I was trying to consider any weird edge-cases that might occur.